jest.conf/setup.visual.js

```diff
@@ -0,0 +1,72 @@
+import 'regenerator-runtime/runtime';
+import '@testing-library/jest-dom';
+import * as Aphrodite from 'aphrodite';
+import * as AphroditeNoImportant from 'aphrodite/no-important';
+
+// eslint-disable-next-line import/no-unresolved
```

Comment:
I had thought about that initially. Will make the changes soon 👍🏻

Reply:
Would it be okay if I created separate setup files for unit and visual tests and put all the common setup configuration in the original setup.js and import it in the test specific setup files?

Reply:
What are the differences between the two? @KshitijThareja Is there something in setup.js that we dont need in visual.setup.js?

Reply:
There's the `window property` definition and the` TEST_TYPE` env var declaration that's present in setup.js which is changed in the visual.setup.js. Simply importing it would cause errors in the testing environment.

Reply:
You mean the `scrollTo` override?
```JS
Object.defineProperty(window, 'scrollTo', { value: () => {}, writable: true });
```
What could be the problem with this? And the Test type wouldnt be a problem because at the end if we import 'setup.js' we will be excecuting that code earlier, and then we can just override it in the `visual.setup.js`. But if the scrollTo override is a big problem, we will definetely need to separate them in different files

Reply:
The window property is only defined for jsdom environments. Since we are using a node environment defined as a part of the jest-puppeteer package, it will cause the following error:

> ● Test suite failed to run
> 
>     The error below may be caused by using the wrong test environment, see https://jestjs.io/docs/configuration#testenvironment-string.
>     Consider using the "jsdom" test environment.

Reply:
In our regular check-in meetings we saw that we could add a type check of `window` so we can reuse this file :open_hands:.